docs: add section about milestones to contributing (#485)
solid-design-system/solid: CONTRIBUTING.md

- We optimize our components for accessibility.
- We don't mention the names of internal or external colleagues in issues or other documents hosted on GitHub, as our project is entirely public and can be accessed by anybody. Tagging/mentioning colleagues using their GitHub profiles is fine, as they decided to be visible on GitHub.

### Milestones

We use milestones as a way to group issues and pull requests together towards a specific goal we want to achieve.

Milestones have:

> - A name which defines to which goal the milestone is working towards.

Author: ```suggestion
- A name which shortly defines to which goal the milestone is working towards.
```

> - A start date which defines when the milestone starts.

Author: ```suggestion
```

- An end date which defines when the milestone ends.

> - A description which defines what the milestone is about.

Author: ```suggestion
- A description which provides more detailed information about what we want to achieve in this milestone.
```
